src/main/java/com/incentives/piggyback/partner/dto/PartnerOrderEntity.java

```diff
@@ -26,4 +27,7 @@
 	private Date createdDate;
 	private Date lastModifiedDate;
 	private Integer isActive;
+	private List<String> interestCategories;
+	private String vendorDisplayName;
+	private String vendorRedirectUrl;
```

Comment:
VendorDisplayName can be inferred. PartnerOrder contains partnerId, and partnerId can provide partnerName from Partner object.

Also, what is vendorRedirectUrl? If it is the webhook address, we have partnerWebHookAddress in the Partner object.